release/models/gnsi/openconfig-gnsi-acctz.yang

```diff
@@ -0,0 +1,245 @@
+module openconfig-gnsi-acctz {
+  yang-version 1;
+  namespace "https://github.com/openconfig/yang/gnsi/acctz";
+  prefix oc-gnsi-acctz;
+
+  import openconfig-extensions { prefix oc-ext; }
+  import openconfig-system {
+    prefix oc-sys;
+  }
+  import openconfig-system-grpc {
+    prefix oc-sys-grpc;
+  }
+  import openconfig-types {
+    prefix oc-types;
+  }
+  import openconfig-yang-types {
+    prefix oc-yang;
+  }
+  import openconfig-gnsi {
+    prefix oc-gnsi;
+  }
+  organization
+    "OpenConfig Working Group";
+
+  contact
+    "OpenConfig working group
+    netopenconfig@googlegroups.com";
+
+  description
+    "This module provides a data model for the metadata of the gRPC
+    accounting operations on a device.";
```

Comment:
This module provides counters of gNSI accountZ  requests and responses and the quantity of data transferred.